ui/component/userVerify/view.jsx

```diff
@@ -73,7 +73,7 @@ class UserVerify extends React.PureComponent<Props> {
                 />
                 <p className="help">
                   {__('Standard messaging rates apply. LBRY will not text or call you otherwise. Having trouble?')}{' '}
-                  <Button button="link" href="https://lbry.com/faq/phone" label={__('Read more.')} />
+                  <Button button="link" href="https://lbry.com/faq/phone" label={__('Read more')} />.
```

Comment:
Why not make it 'Read More' (for consistency), and remove the duplicate 'Read more ' from strings?

Reply:
for future reuse e.g. 
```
<token:button:%read_more%>
"%read_more% or refresh the page to fix it."
```

Reply:
OK. then make it here 'Read more.' (It's a sentence) and leave also the 'Read more' in string list.

Reply:
But where did you find  this? I can't find it.
<token:button:%read_more%>
"%read_more% or refresh the page to fix it."

Also I searched for this strings below and I can't find them anywhere.
 "%duration% minute ago": "%duration% minute ago",
  "%duration% seconds ago": "%duration% seconds ago",
  "%duration% second ago": "%duration% second ago",
  "%duration%  ago": "%duration%  ago",

Reply:
As I said, "future, e.g.".

As for %duration%, the string ID is generated on runtime.  Search for `%duration%`, or delete it on your local runtime and see it being automatically added.

Reply:
OK, also there are some strings like "Earn %rewards_link% for inviting your friends. Read our %referral_faq_link% to learn more." from "ui\component\inviteNew\view.js" that for some reasons are not translated. Everything seems fine in code, the text is already there in Transifex translated, but on the app interface is not translated. Maybe you can spot what could be the cause. I can't find anything wrong.

Reply:
Ah, I've fixed that bug earlier this week, and Sean has merged into the next RC. There was a mistake in the `<i18nMessage>` that only translated the tokens and not the entire sentence.